assume/strategies/learning_strategies.py

```diff
@@ -178,8 +178,8 @@ def calculate_bids(
         ]
 
         # store results in unit outputs which are written to database by unit operator
-        unit.outputs["rl_observations"][start] = next_observation
-        unit.outputs["rl_actions"][start] = actions
+        unit.outputs["rl_observations"].append(next_observation)
+        unit.outputs["rl_actions"].append(actions)
         unit.outputs["rl_exploration_noise"][start] = noise
```

Comment:
This is differen (append vs [start]) because you commented it out right? Shoudl we comment everything out then?

Reply:
I don't understand this point, please clarify

Reply:
Nevermind, not thta important